source/_integrations/ezviz.markdown

```diff
@@ -17,231 +19,11 @@ It also exposes an RTSP stream, by using the local camera IPs (so the device hos
 
 As there is no official documentation for the API, the component retrieves data from the API used in the Ezviz mobile app, [hosted here](https://apiieu.ezvizlife.com).
 
-## Configuration
-
-The configuration (see below) needs your Ezviz username & password you use within the Ezviz mobile application.
-
-Next, add the Ezviz camera to your `configuration.yaml` file like below:
-
-```yaml
-# Example configuration.yaml entry
-camera:
-  - platform: ezviz
-    username: YOUR_USERNAME
-    password: YOUR_PASSWORD
-    cameras:
-      D12345678:
-        username: YOUR_CAMERA_USERNAME
-        password: YOUR_CAMERA_PASSWORD
-```
-
 As you see, here is the way to describe the camera credentials.
 The password for each camera is usually written underneath the device, near the QR code. It is also code "verification code".
```

Comment:
This paragraph refers to "as you see", but the thing is referencing is not there?